src/wallet/rpcwallet.cpp

```diff
@@ -1299,6 +1300,12 @@ UniValue addwitnessaddress(const JSONRPCRequest& request)
         throw std::runtime_error(msg);
     }
 
+    if (!IsDeprecatedRPCEnabled("addwitnessaddress")) {
+        throw JSONRPCError(RPC_METHOD_DEPRECATED, "addwitnessaddress is deprecated and will be fully removed in v0.17. "
+            "To use estimatefee in v0.16, restart bitcoind with -deprecatedrpc=estimatefee.\n"
```

Comment:
This still mentions estimatefee :)

Reply:
Also, if you deprecate it like this you'll have to add -deprecatedrpc=addwitnessaddress to the tests which use it (easier than reworking the tests at the same time)

Reply:
Gah, will update. Nothing is ever easy, is it? I assumed I was just updating the documentation hence I didn't bother running any tests.